jwt-java-client-okhttp/src/test/java/com/clouway/oauth2/client/IssueJwtTokensTest.java

```diff
@@ -84,19 +85,27 @@ public void issueNewToken() throws IOException, NoSuchAlgorithmException, Invali
     assertThat(possibleToken.get().isAvailableAt(new Date()), is(true));
   }
 
-  @Test(expected = IOException.class)
+  @Test
   public void gotUnexpectedResponse() throws IOException, NoSuchAlgorithmException, InvalidKeySpecException {
-    clientDriver.addExpectation(
-            onRequestTo("/o/oauth2/v1/token").withMethod(Method.POST),
-            giveJsonResponse(ImmutableMap.<String, Object>of("error", "invalid_grant")).withStatus(400)
-    );
-
-    JwtConfig config = new Builder("2000001@apps.telcongserviceaccount.com", clientDriver.getBaseUrl() + "/o/oauth2/v1/token", PRIVATE_KEY.getBytes())
-            .subject("someuser@clouway.com")
-            .build();
-
-    TokenSource tokenSource = config.tokenSource();
-    tokenSource.token(new Date());
+    try {
+      clientDriver.addExpectation(
+              onRequestTo("/o/oauth2/v1/token").withMethod(Method.POST),
+              giveJsonResponse(ImmutableMap.of("error", "invalid_grant")).withStatus(400)
+      );
+
+      JwtConfig config = new Builder("2000001@apps.telcongserviceaccount.com", clientDriver.getBaseUrl() + "/o/oauth2/v1/token", PRIVATE_KEY.getBytes())
+              .subject("someuser@clouway.com")
+              .build();
+
+      TokenSource tokenSource = config.tokenSource();
+      tokenSource.token(new Date());
+      
+      fail("IOException was not thrown when token endpoint fails?");
```

Comment:
Message is misleading for me. I think the question mark is the reason. I think just "IOException was not thrown on token endpoint failure." sounds better